performance/compare.py

```diff
@@ -5,6 +5,97 @@
 except ImportError:
     import json
 
+from math import sqrt
+from statistics import stdev, mean
+#import scipy.stats as st
+# scipy.stats.norm.ppf(0.9)
```

Comment:
Do we need any of this commented-out code?

Reply:
Yes - it's the notes on how we came up with the Z-score threshold used for coloring.